x-pack/plugins/security_solution/public/management/cypress/tasks/login.ts

```diff
@@ -48,8 +48,13 @@ export const login: CyLoginTask = (
 ): ReturnType<typeof sendApiLoginRequest> => {
   let username = Cypress.env('KIBANA_USERNAME');
   let password = Cypress.env('KIBANA_PASSWORD');
+  const isServerless = Cypress.env('IS_SERVERLESS');
+  const isCloudServerless = Cypress.env('CLOUD_SERVERLESS');
 
-  if (user) {
+  if (isServerless && isCloudServerless) {
+    // MKI QA Cloud Serverless
+    return sendApiLoginRequest(username, password);
```

Comment:
For MKI environments it is expected to perform the login through SAML not API.

Reply:
Switched to SAML auth for MKI, "regular" serverless login will be aligned in next iterations since it wouldn't work oob.